src/vdbfusion/vdbfusion/VDBVolume.h

```diff
@@ -41,16 +41,18 @@ class VDBVolume {
     /// @brief Integrates a new (globally aligned) PointCloud into the current
     /// tsdf_ volume.
     void Integrate(const std::vector<Eigen::Vector3d>& points,
+                   const std::vector<Eigen::Vector3d>& colors,
                    const Eigen::Vector3d& origin,
                    const std::function<float(float)>& weighting_function);
 
     /// @brief Integrates a new (globally aligned) PointCloud into the current
     /// tsdf_ volume.
     void inline Integrate(const std::vector<Eigen::Vector3d>& points,
+                          const std::vector<Eigen::Vector3d>& colors,
```

Comment:
As is its here, `colors` is not optional, right? Shall we add an overload function, or default parameters just to make it "non-color" compatible, like the case for LiDARs

Reply:
It is not optional but it works if an empty vector is provided. I can add a default, but then it will be moved and be later in the list I think. Might be better to just add another function that doesn't have the color argument, C++ should be able to deal with that.

Reply:
Done.